.mci.yml

```diff
@@ -385,6 +400,16 @@ tasks:
         - func: "fetch mongodb"
         - func: "run integration tests"
 
+    - name: "integration test 3.0 with WiredTiger"
+      depends_on:
+        - name: "compile"
+      commands:
+        - func: "set version 3.0"
+        - func: "use WiredTiger storage"
+        - func: "fetch artifacts"
+        - func: "fetch mongodb"
+        - func: "run integration tests"
+
     - name: "integration test latest"
```

Comment:
Same idea: I don't want to specify what latest uses by default: I just want whatever the default is.